source/configuration_manual/mail_compress_plugin.rst

```diff
@@ -42,8 +42,7 @@ or .bz2 extension. There is no support for compression during saving.
 -----------------------
 
 Mails can be stored as compressed. Existing uncompressed mails can't currently
-be directly compressed (or vice versa). You could, however, use `dsync
-<https://wiki.dovecot.org/Tools/Doveadm/Sync>`_
+be directly compressed (or vice versa). You could, however, use :man`dsync(1)`
```

Comment:
Broken link, i.e. missing colon typo.

Reply:
This is giving an error still: `undefined label: man-1_dsync`. Should this be: ```:man:`doveadm-sync(1)` ```?